specification/redis/Cache.Management/RedisCacheAccessPolicyAssignment.tsp

```diff
@@ -0,0 +1,70 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Cache;
+/**
+ * Response to an operation on access policy assignment
+ */
+@parentResource(RedisResource)
+model RedisCacheAccessPolicyAssignment
+  is Azure.ResourceManager.ProxyResource<RedisCacheAccessPolicyAssignmentProperties> {
+  ...ResourceNameParameter<
+    Resource = RedisCacheAccessPolicyAssignment,
+    KeyName = "accessPolicyAssignmentName",
+    SegmentName = "accessPolicyAssignments",
+    NamePattern = "^([a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|[a-zA-Z0-9])$"
+  >;
+}
+
+@armResourceOperations
+interface RedisCacheAccessPolicyAssignments {
+  /**
+   * Gets the list of assignments for an access policy of a redis cache
+   */
+  get is ArmResourceRead<RedisCacheAccessPolicyAssignment>;
+
+  /**
+   * Adds the access policy assignment to the specified users
+   */
+  createUpdate is ArmResourceCreateOrReplaceAsync<
+    RedisCacheAccessPolicyAssignment,
+    LroHeaders = ArmLroLocationHeader & Azure.Core.Foundations.RetryAfterHeader
+  >;
+
+  /**
+   * Deletes the access policy assignment from a redis cache
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-delete-operation-response-codes" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  delete is ArmResourceDeleteWithoutOkAsync<
```

Comment:
Does this get grandfathered in as backcompat, or do you leave the fixme for us to consider fixing in future versions?

Reply:
Should just be DeleteAsync to avoid response customization